src/lighteval/utils/tokenizers.py

```diff
@@ -0,0 +1,361 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
We've tried to keep `utils` for generic logic items, so I wonder if tokenizers should not go elsewhere - no strong opinion but would welcome @NathanHB 's ideas on this

Reply:
This should go in the metrics !

Reply:
MOved and Renamed for better semantics :)